tenant/pkg/util/tenant_utils.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	tenancyv1alpha1 "github.com/multi-tenancy/tenant/pkg/apis/tenancy/v1alpha1"
+)
+
+// Determine the tenant namespace name based on prefix requirement
```

Comment:
Maybe worth documenting this a bit more? I'm not sure what the "prefix requirement" is - when should it be true or false?

Also maybe use the godoc standard since this is now an exported function?

Reply:
I am not aware of godoc standard, do you have any sample for me to follow?

Reply:
Just start the comment with the name of the function (or field, etc):

```
// GetTenantNamespace determines the name of the namespace ...
```

Reply:
done